package.json

```diff
@@ -101,6 +101,7 @@
     "verifyConditions": "condition-circle"
   },
   "engines": {
-    "node": ">=14.x"
+    "node": ">=16.11",
```

Comment:
@crecotun  What was the reason to enforce 16.11 here?
pex-client is stuck on 16.10 for the moment because of https://github.com/facebook/jest/issues/11956

Reply:
@tzimmermann npm 8 landed in versions starting from 16.11, that was the main reason
Actually, I'd not expect a module from npm affecting your build system, as you're using an artifact 🤔

Reply:
Yarn is still validating the `engines` in all `package.json`s apparently. You can see it failing here: https://app.circleci.com/pipelines/github/signavio/pex-client/36922/workflows/b27da500-cd18-4a46-a3cf-3fe1e9e0f9f6/jobs/125830

Reply:
One could apparently ignore this check by using a flag on yarn install: https://classic.yarnpkg.com/lang/en/docs/cli/install/#toc-yarn-install-ignore-engines
But I am also wondering why it didn't error on installing locally in the first place, but just on CI 🤔